### PR TITLE
Math magicians: full website

### PR DESCRIPTION
**Pull request by waelelsafty07**
## PR Details

- Create a website consisting of 3 pages: Home, Calculator, and Quote.
- For every page, you should create a <Route>.
- Navigate links for all 3 routes are displayed on each page.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,12 +1,23 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (CollinsTatang) [inline src/App.js]: - [ ] Please do not disable your `eslint` linter check
Kindly fix all your linters errors to help you write standard code

File: src/components/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```
Comment (CollinsTatang) [inline src/components/Navbar.js]: - [ ] Please do not disable your `eslint` linter check
Kindly fix all your linters errors to help you write standard code

**Review by CollinsTatang — CHANGES_REQUESTED** [review]
Hi @waelelsafty07 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

### **Highlights 🚀 :heavy_check_mark: :heavy_check_mark:** 

- The repo looks professional.:heavy_check_mark:
- Git/GitHub flow best practices have been followed:heavy_check_mark:
- There are no linter errors:heavy_check_mark:
- You have descriptive PR:heavy_check_mark:

## Required changes 
It's ok to have bugs in your code. Please, see this code review as a chance for you to learn practical coding skills and learn new tips that can improve your code quality and help you get a better job.
_Check the comments under the review._
## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @CollinsTatang in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by jerryowusu — APPROVED** [review]
Hi @waelelsafty07 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

chelsea-mason-approve


## Highlights
Used react-router-dom :heavy_check_mark: 
Made requested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**